var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -51,8 +51,8 @@ class Mfem(Package):
     # If this quick verification procedure fails, additional discussion
     # will be required to verify the new version.
 
-    version('3.3.1-rc1', git='https://github.com/mfem/mfem',
-            tag='v3.3.1-rc1')
+    version('3.3.2-rc2', git='https://github.com/mfem/mfem',
+            tag='v3.3.2-rc2')
```

Comment:
Please keep the older version and use the new version() with checksum:
```
version("v3.3.2-rc2", "<CHECKSUM>", url='https://github.com/mfem/mfem/archive/v3.3.2-rc2.tar.gz')
```

Reply:
1. I don't see the 3.3.1-rc1 so its best to replace

2. is the checksum required here [wrt url to git tag] ? If so we'll have to revise xsdk workflow instructions.

Reply:
@junghans: do we need to require checksummed versions of release candidates that are not out yet?  In this case the xSDK folks are preparing a release, and these additions are mostly so that people can try out the experimental release candidates.  Note that the latest stable version is checksummed and preferred, so no one will get this release w/o being explicit about it.

Reply:
@belay, are you moving git tags? If so, let's keep it like it is. Otherwise, I just prefer having a checksum, so that I know, I got the right thing, when downloading code.

Reply:
@acfisher might know about the rc tags - and what would happen to them after the mfem release

Reply:
I guess the other thing we could do would be handle rc tags only on the xsdk release branch, to avoid churn on Spack `develop`, and only submit final releases to `develop`.  That kind of complicates the workflow, though, and if package devs want others to try their release candidates, they'll need to know about a branch.

Reply:
We will likely remove the RC tags after the release and just stamp the final version v3.3.2.  The 3.3.1 tag was in error as we only use even numbers for release candidates.